packages/manager/modules/pci/src/projects/project/data-processing/notebooks/notebook-details/translations/Messages_fr_FR.json

```
@@ -0,0 +1,24 @@
+{
+  "data_processing_notebooks_details_dashboard_label": "Tableau de bord",
+  "data_processing_notebooks_details_information_label": "Information générales",
```

```suggestion
  "data_processing_notebooks_details_information_label": "Informations générales",
```
If you choose to put a plurial for "générales", I think you can add an other one for "informations".

---

Fixed :)